packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -13,6 +13,8 @@ export enum FieldType {
   number = 'number',
   string = 'string',
   boolean = 'boolean',
+  // Used to detect that the value is some kind of trace data to help with the visualisation and processing.
+  trace = 'trace',
```

Comment:
what does a trace field mean? is it a number? should this not be just a DataFrame meta property or label instead?

Reply:
I assume it means it is a JSON blob with trace info...  will be more interesting/obvious with something like #22083

Reply:
Does it mean the cell value is an object that implements:
https://github.com/grafana/grafana/blob/add-trace-ui/packages/jaeger-ui-components/src/types/trace.tsx#L77

Reply:
@ryantxu Yes basically that is right. It just wraps the response from jaeger into dataframe.fields[0].values[0] and that's it. Not sure if the best way to do this. Other options could be to have different queryTrace method on dataframe or extend possible return types in to include something like trace.

At the same time thought theoretically DS could return multiple traces (not sure if possible in Jaeger but for example to compare traces or do some aggregation over spans it could make sense) so maybe dataframe is not that weird for this.